csv_generator/csv_generator.py

```diff
@@ -0,0 +1,63 @@
+import json_to_csv_converter
+import pandas as pd
+import os
+
+__dir__ = "data" # modify dataset directory if needed
+__location__ = os.path.realpath(
+    os.path.join(os.getcwd(), __dir__)
+)
+
+class CsvGenerator():
+    
+    def __init__(self, business_filename: str, review_filename: str):
+        self.business_filename = business_filename
+        self.review_filename = review_filename
+        self.business_ids = set()
+        
+    def read_csv(self):
+        business_df = pd.read_csv(os.path.join(__location__, self.business_filename))
```

Comment:
Could pull `os.path.join(__location__, filename)` into a util function in a separate file since it's being used multiple times.

eg filepath_util.py
```
def get_os_path(filename):
  os.path.join(__location__, filename) # also add the __location__ in this file
```